### ISSUE TITLE
Are there any meaningful performance differences in assemblies for any of these choices?

### ISSUE THREAD

**Issue by StingyJack** [MicrosoftDocs/visualstudio-docs#1355]
"Please fill out this field"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 56f58310-fec4-2374-f9b1-253ae98fccf9
* Version Independent ID: 38ebce67-8953-ce1b-a082-e39c936d5865
* Content: [Advanced Build Settings Dialog Box (C#) - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/reference/advanced-build-settings-dialog-box-csharp)
* Content Source: [docs/ide/reference/advanced-build-settings-dialog-box-csharp.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/reference/advanced-build-settings-dialog-box-csharp.md)
* Product: **visual-studio-dev15**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by cartermp** [comment]
Yes, selecting a different version of the C# language can result in different assemblies with meaningful performance differences  _provided_ that you also use features that affect this. For example, if you have a C# 7.0 project and upgrade to C# 7.3, then make good use of the `Span<'T>` feature, your code can run faster and allocate less memory.

**Comment by ScottHutchinson** [comment]
But what are the performance differences (if any) between the different debug info settings (none, pdbonly, or full)? In particular, what performance impact would there be on a *release* build with pdbonly or full debug info?
